CRM/Export/BAO/Export.php

```diff
@@ -749,7 +710,7 @@ public static function exportCustom($customSearchClass, $formValues, $order) {
       $rows[] = $row;
     }
 
-    CRM_Core_Report_Excel::writeCSVFile(self::getExportFileName(), $header, $rows);
+    CRM_Core_Report_Excel::writeCSVFile(ts('CiviCRM Contact Search'), $header, $rows);
     CRM_Utils_System::civiExit();
```

Comment:
Not sure about this change as it duplicates a string - do we need to make this change?

Reply:
The point is to stop it calling what is essentially an unrelated function for a different export process - ie. to break dumb interdependencies

Reply:
Right. Totally makes sense then :-)